rita_common/src/rita_loop/mod.rs

```diff
@@ -75,8 +76,8 @@ pub fn start_core_rita_endpoints(workers: usize) {
 
 pub fn check_rita_common_actors() {
     assert!(crate::hello_handler::HelloHandler::from_registry().connected());
-    assert!(crate::traffic_watcher::TrafficWatcher::from_registry().connected());
     assert!(crate::rita_loop::fast_loop::RitaFastLoop::from_registry().connected());
     crate::rita_loop::slow_loop::start_rita_slow_loop();
     crate::rita_loop::fast_loop::start_rita_fast_loop();
+    init_traffic_watcher();
```

Comment:
this needs to go above the fast/slow loop start statements, otherwise it's a race condition, where the fast loop traffic watcher could panic since the counters are not installed.